Revision Machine tests failing when executed on machines with non-CEST compatible timezones

Not sure if I have something misconfigured, but just pulled the most recent master to work on #159 and when trying to install `dkpro-jwlp` I got the following error in the tests:

```
[ERROR] Failures:
[ERROR]   RevisionApiTest.articleIDTests:183 expected:<2004-04-07 0[2]:31:34.0> but was:<2004-04-07 0[1]:31:34.0>
[INFO]
[ERROR] Tests run: 7, Failures: 1, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] JWPL ............................................... SUCCESS [  4.769 s]
[INFO] Java Wikipedia API ................................. SUCCESS [ 28.459 s]
[INFO] MediaWiki Dumper ................................... SUCCESS [  0.917 s]
[INFO] WikiMachine ........................................ SUCCESS [  0.287 s]
[INFO] DataMachine ........................................ SUCCESS [  3.122 s]
[INFO] TimeMachine ........................................ SUCCESS [  2.473 s]
[INFO] RevisionMachine .................................... FAILURE [  5.222 s]
[INFO] MediaWiki Parser ................................... SKIPPED
[INFO] JWPL API and RevisionAPI Utilities ................. SKIPPED
[INFO] JWPL Tutorial ...................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 45.474 s
[INFO] Finished at: 2018-07-25T11:03:59+01:00
[INFO] Final Memory: 120M/1222M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.21.0:test (default-test) on project de.tudarmstadt.ukp.wikipedia.revisionmachine: There are test failures.
[ERROR]
[ERROR] Please refer to /Users/thiago/code/tgalery/dkpro-jwpl/de.tudarmstadt.ukp.wikipedia.revisionmachine/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
```

Is someone able to replicate it ? Why isn't travis picking that up ? Should I tackle as part of #159 ?

--- a/.gitignore
+++ b/.gitignore
@@ -16,3 +16,5 @@ target
 *.tmp
 *_hypoCountMap
 
+### Embedded DB properties file
+de.tudarmstadt.ukp.wikipedia.revisionmachine/src/test/resources/db/*.properties

--- a/de.tudarmstadt.ukp.wikipedia.revisionmachine/src/test/java/de/tudarmstadt/ukp/wikipedia/revisionmachine/RevisionApiTest.java
+++ b/de.tudarmstadt.ukp.wikipedia.revisionmachine/src/test/java/de/tudarmstadt/ukp/wikipedia/revisionmachine/RevisionApiTest.java
@@ -24,6 +24,8 @@ import static org.junit.Assert.fail;
 import java.lang.reflect.Field;
 import java.sql.SQLException;
 import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
 import java.util.Calendar;
 
 import org.junit.*;
@@ -38,6 +40,12 @@ import de.tudarmstadt.ukp.wikipedia.revisionmachine.api.RevisionApi;
 public class RevisionApiTest extends BaseJWPLTest {
 
 	private static Wikipedia wiki = null;
+
+	private Timestamp convertToUTC(Timestamp ts) {
+
+		final LocalDateTime dt = LocalDateTime.ofInstant(ts.toInstant(), ZoneOffset.UTC);
+		return Timestamp.valueOf(dt);
+	}
 
 	// The object under test
 	private RevisionApi revisionApi;
@@ -176,12 +184,12 @@ public class RevisionApiTest extends BaseJWPLTest {
 		try {
 			int pageId = wiki.getPage("Car").getPageId();
 
-			Timestamp firstDayOfAppearance = revisionApi.getFirstDateOfAppearance(pageId);
-			Timestamp lastDayOfAppearance = revisionApi.getLastDateOfAppearance(pageId);
+			Timestamp firstDayOfAppearance = convertToUTC(revisionApi.getFirstDateOfAppearance(pageId));
+			Timestamp lastDayOfAppearance = convertToUTC(revisionApi.getLastDateOfAppearance(pageId));
 			int nrOfRevisions = revisionApi.getNumberOfRevisions(pageId);
 
-			assertEquals("2004-04-07 02:31:34.0", firstDayOfAppearance.toString());
-			assertEquals("2009-01-19 04:58:09.0", lastDayOfAppearance.toString());
+			assertEquals("2004-04-07 00:31:34.0", firstDayOfAppearance.toString());
+			assertEquals("2009-01-19 03:58:09.0", lastDayOfAppearance.toString());
 			assertEquals(382, nrOfRevisions);
 		}
 		catch (WikiApiException e) {

--- a/de.tudarmstadt.ukp.wikipedia.revisionmachine/src/test/resources/db/wikiapi_simple_20090119_stripped.properties
+++ b/de.tudarmstadt.ukp.wikipedia.revisionmachine/src/test/resources/db/wikiapi_simple_20090119_stripped.properties
@@ -1,5 +1,0 @@
-#HSQL Database Engine 2.4.0
-#Tue Jul 17 10:22:21 CEST 2018
-version=2.4.0
-modified=yes
-tx_timestamp=0

